Templates/MonoGame.Templates.VSExtension/MonoGameTemplatesVSExtensionPackage.cs

```diff
@@ -25,6 +28,7 @@ namespace MonoGame.Templates.VSExtension
     /// </remarks>
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid(MonoGameTemplatesVSExtensionPackage.PackageGuidString)]
+    [ProvideEditorExtension(typeof(EditorFactory), ".mgcb", int.MaxValue)]
```

Comment:
If I read the docs correctly, there should be a `Default Name` property that can be used to specify the name of the editor when you right click the file.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.provideeditorextensionattribute.defaultname?view=visualstudiosdk-2022